### ISSUE TITLE
There is a logic error around dateTimeFns.since() and negative duration values

### ISSUE THREAD

**Issue by kolton-musgrove** [rubber-duck-software/iso-fns#63]
```
describe('.since()', () => {
  const dt = dateTimeFns.from('1976-11-18T15:23:30.123')
  it('dateTimeFns.since(dt, earlier) == dateTimeFns.until(earlier, dt)', () => {
    const earlier = dateTimeFns.from({ year: 1966, month: 3, day: 3, hour: 12 })
    const datetime1 = dateTimeFns.since(dt, earlier)
    const datetime2 = dateTimeFns.until(earlier, dt)
    assert.equal(datetime1, datetime2)
  })
```

The values of datetime1 and datetime2 are different even though they are logically equivalent.

**Comment by kolton-musgrove** [comment]
This seems to affect a variety of tests and is the result of a rounding or truncation error that gets the value wrong by two days on a consistent basis.

Not sure where this is coming from though. I would guess ES.DifferencceISODateTime or ES.RoundDuration inside of DateTimeFns.ts or DifferenceISODateTime inside of ecmascript.ts

**Comment by kolton-musgrove** [comment]
Upon further investigation, this is related to the handling of negative hour overflow.

**Comment by rubber-duck-software** [comment]
This is resolved in 2.0.0-alpha.8
